app-misc/jigdo/jigdo-0.7.3.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Ease the distribution of files"
+DESCRIPTION="Jigsaw Download, or short jigdo, is a tool designed to ease the distribution \
+of very large files over the internet, for example CD or DVD images. Its aim is \
+to make downloading the images as easy for users as a click on a direct \
+download link in a browser, while avoiding all the problems that server \
+administrators have with hosting such large files."
+HOMEPAGE="http://atterer.org/jigdo/"
+COPYRIGHT="2001-2006 Richard Atterer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://atterer.org/sites/atterer/files/2009-08/jigdo/jigdo-$portVersion.tar.bz2"
+CHECKSUM_SHA256="875c069abad67ce67d032a9479228acdb37c8162236c0e768369505f264827f0"
+PATCHES="jigdo-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	jigdo$secondaryArchSuffix = $portVersion
+	cmd:jigdo$secondaryArchSuffix = $portVersion
+	cmd:jigdo_file$secondaryArchSuffix = $portVersion
+	cmd:jigdo_lite$secondaryArchSuffix = $portVersion
+	cmd:jigdo_mirror$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libdb$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libdb$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:wget$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	./configure --bindir=$binDir --mandir=$manDir --datadir=$dataDir
```

Comment:
You may wish to use [`runConfigure`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#runconfigure) instead

Reply:
@alaviss Yes, I have thought of this before but `runConfigure` will not work in this case. There will be an unrecognized option error which is `docdir` and `datarootdir`. I tried using `--omit-dirs docdir datarootdir` but it seems like it `datarootdir` cannot be omitted (`datarootdir: command not found`).

Reply:
`--omit-dirs` only takes one argument. What you want is `--omit-dirs "docDir dataRootDir"`

See example: [avra-1.3.0.recipe](https://github.com/haikuports/haikuports/blob/master/dev-embedded/avra/avra-1.3.0.recipe#L35)

Reply:
@alaviss Ah, I see. Thanks for the info. 👍